apps/site-landing/src/components/RangeFieldAndScale/index.tsx

```diff
@@ -20,52 +20,52 @@ export type Props = {
 }
 
 export const RangeFieldAndScale = (props: Props) => {
+  const [value, setValue] = useState(props.initial ?? 0);
 
-    const [value, setValue] = useState(props.initial ?? 0);
-
-    const minimum = props.minimum ?? 0;
-    const maximum = props.maximum;
-    const step = props.step ?? 1;
+  const minimum = props.minimum ?? 0;
+  const { maximum } = props;
+  const step = props.step ?? 1;
 
-    const onChange = ([value]: number[]) => {
-      setValue(value);
-      props.onChange(value);
-    }
+  const onChange = ([value]: number[]) => {
+    setValue(value);
+    props.onChange(value);
+  };
 
-    return (
-      <div className={styles.container}>
-        <div className={styles.label}>
-          <FormattedMessage {...props.label} tagName="span" />
-          <FormattedNumber
-            value={value}
-            currency={props.currency}
-            currencyDisplay={"narrowSymbol"}
-            style={props.scaleType}
-            unit={props.unit}
-            minimumFractionDigits={0}
-            maximumFractionDigits={0}
-           />
-        </div>
-        <Range
-          step={step}
-          min={minimum}
-          max={maximum}
-          values={[value]}
-          onChange={onChange}
-          renderTrack={(props) => (
-            <SliderTrack
-              value={value}
-              max={maximum}
-              min={minimum}
-              {...props}
-            />)}
-          renderThumb={SliderThumb}
+  return (
+    <div className={`${styles.container} x6spaceBefore`}>
```

Comment:
In my merge, I've moved this x6spaceBefore back into the owning (`<Basic />`) class.  I need to squeeze a lot more sliders into the advanced pane and this much margin is top of the chopping block :-).